app/assets/v2/js/pages/profile_tokens.js

```diff
@@ -1,5 +1,7 @@
-const factoryAddress = '0x7bE324A085389c82202BEb90D979d097C5b3f2E8';
-const DAI = '0x6b67DD1542ef11153141037734D21E7Cbd7D9817';
+// Personal token constants
+// Note that this address is also duplicated in board.js
+const factoryAddress = '0x80D50970599E33d0D5D436A649C25b729666A015';
+const defaultTokenName = 'DAI';
```

Comment:
Same as above, perhaps we call this `purchaseTokenName`

Reply:
Updated in b4ec2ce